docs/insomnia/custom-themes.md

```diff
@@ -14,25 +14,33 @@ Make a custom Insomnia theme by creating an Insomnia [plugin](/insomnia/introduc
 
 1. Create a new project for your theme. If you plan to make this theme available via NPM, start your project name with `insomnia-plugin-`. The following example is called `insomnia-plugin-dark-colorblind-theme`.
 
-2. Following plugin instructions, write your [plugin package.json](https://docs.insomnia.rest/insomnia/introduction-to-plugins#plugin-packagejson). In your entry file, export your theme.
+2. Following plugin instructions, write your [plugin package.json](https://docs.insomnia.rest/insomnia/introduction-to-plugins#plugin-packagejson). In your entry file, export your theme(s).
 
-3. Start with a baseline template. Each section of the configuration has `background` and `foreground` options, with seven (7) modifiable color options. 
-   
-   See the code comments for more information about which color option names correspond with which UI elements.
+3. Start with a baseline template. Each section of the configuration has `background`, `foreground`, and `highlight` options, with modifiable color options.
+
+  See the code comments for more information about which color option names correspond with which UI elements.
+
+  {:.alert .alert-primary}
+  **Note**: The code comments below are not comprehensive and styles may apply elsewhere.
 
 ```ts
+// If you want to generate your plugin from your entry file, 
+// use module.exports.themes = [{ ... }] instead. 
```

Comment:
```suggestion
// If you want to export your theme from the entry file, 
// use module.exports.themes = [{ ... }] instead. 
```

I do think it would be better for this not to be a comment, however. Like the other plugin examples, it should be possible for a user to copy-paste the entire example snippet directly into the `main.js` file that is created by the generate plugin button from inside the app. This is the least friction, as we don't assume a directory structure with the examples in docs

Reply:
Thank you for updating this one 🙌